scss/detail.scss

```diff
@@ -93,5 +93,97 @@
         }
       }
     }
+  }  
+  @media only screen and (max-width: $flex-mobile) {
+    height: 667px;
+    width: 100%;
+    flex-direction: column-reverse;
+  
+    .return-button {
+      align-self: center;
+      font-size: 18px;
+      line-height: 24px;
+      margin: 20px 0 0 0;
+
+      &::before {
+        margin-right: 12px;
```

Comment:
This rule is the same as in the base class, so you can omit redefining this selector